abis_mapping/utils/checks.py

```diff
@@ -0,0 +1,109 @@
+"""Provides extra frictionless checks for the package"""
+
+
+# Third-Party
+import frictionless
+
+# Local
+from . import coords
+
+# Typing
+from typing import Any, Iterator
+
+
+class ValidCoordinates(frictionless.Check):
+    """Checks whether the latitude and longitude of the row are valid."""
+
+    # Check Attributes
+    code = "valid-coordinates"
+    Errors = [frictionless.errors.RowConstraintError]
+
+    def __init__(
+        self,
+        descriptor: Any=None,
```

Comment:
PEP 8: E252 missing whitespace around parameter equals

Reply:
Fixed in: 4cf2459